node/src/test/scala/com/wavesplatform/mining/BlockV5Test.scala

```diff
@@ -55,6 +58,32 @@ class BlockV5Test
     testTime.advance(offset + 1.milli)
   }
 
+  "Proto block" should "be serialized" in {
+    val features = Seq(534, 3, 33, 5, 1, 0, 12343242).map(_.toShort)
+    val block =
+      TestBlock.create(System.currentTimeMillis(), TestBlock.randomSignature(), Nil, version = Block.ProtoBlockVersion, features = features.sorted)
+    val blockWithBadVotes = block.copy(header = block.header.copy(featureVotes = features))
+
+    Validators.validateBlock(blockWithBadVotes) shouldBe 'left
```

Comment:
Please add checks for all other errors in `validateBlock`: incorrect generator, gen signature, duplicates in votes and etc